src/config_store.c

```diff
@@ -33,15 +37,19 @@ static size_t GetDistance(const ConfigStoreKvpHeader *p, const ConfigStoreKvpHea
 
 size_t ConfigStore_GetKvpFullSize(const ConfigStoreKvpHeader *p, const ConfigStoreKvpHeader *pEnd)
 {
-    if (!p) {
+    if (!p)
```

Comment:
This re-formatting is not in accordance to the style sheet. Did you re-open VS Code after adding the .clang-format file?

Reply:
Resolved